core/plugins/org.polarsys.capella.core.sirius.ui/plugin.xml

```diff
@@ -204,7 +204,22 @@
             </visibleWhen>
          </command>
       </menuContribution>
-         
+       <menuContribution
```

Comment:
Can you move it to the other menu group contributions 
(in https://github.com/eclipse/capella/blob/master/core/plugins/org.polarsys.capella.core.platform.sirius.ui.menu/plugin.xml#L31
)